src/controllers/applicant.ts

```diff
@@ -0,0 +1,46 @@
+import Applicant, { IApplicant } from '../models/applicant';
+
+export const create_post = async (req, res) => {
+    const new_applicant: IApplicant = new Applicant({
+        firstName: req.body.firstName,
+        lastName: req.body.lastName,
+        role: req.body.role,
+        major: req.body.major,
+        yearStanding: req.body.yearStanding,
+        status: req.body.status,
+        linkedIn: req.body.linkedIn,
+        website: req.body.website,
+        resume: req.body.resume,
+    });
+
+    try {
+        await new_applicant.save();
+        res.status(201).send('Successfully created new applicant.');
+    } catch (error) {
+        res.status(500).send(error);
+    }
+};
+
+export const update_status_patch = async (req, res) => {
+    try {
+        const valid_status = ['Pending', 'Accepted', 'Rejected', 'Scheduled'];
+        const my_applicant = await Applicant.findOne({ _id: req.params.id });
+
+        // Check that status is valid
+        if (!valid_status.includes(req.body.status)) {
+            res.status(400).send('"' + req.body.status + '" is not a valid status.');
```

Comment:
There is also literal syntax that makes this much nicer
```
`"${req.body.status}" is not a valid status.
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals